src/app/static/src/tests/root/mutations.test.ts

```diff
@@ -44,6 +45,11 @@ describe("Root mutations", () => {
         router.push('/login')
     })
 
+    afterEach(() => {
+        // restore the spy created with spyOn
+        jest.restoreAllMocks();
```

Comment:
Why `restoreAllMocks` ? 

Should clear mocks before each.

Reply:
From https://dev.to/edwinwong90/jestclearallmocks-vs-jestresetallmocks-vs-jestrestoreallmocks-explained-5aal I think we want to `restoreAllMocks` because it will restore the mocked function from call to `spyOn` back to the original implementation. Whereas `clearAllMocks` just resets the mock usage data. 

I feel like it's generally better to clean up completely after each test to keep each test isolated. Setting a `spyOn` is like modifying global state and that can lead to non-obvious test failures and test ordering issues if not tidied up.

Reply:
Yeah I think makes sense here to use `restoreAllMocks` because we usethe `spyOn` in some tests but not all so from what I understand I think we should clear it up after we create the spy.